packages/babel-plugin/src/styled-macro.js

```diff
@@ -80,36 +79,42 @@ export let styledTransformer = ({
     } else {
       reference.replaceWith(getStyledIdentifier())
     }
+
+    createStyledComponentPath = reference.parentPath
   } else if (
     reference.parentPath &&
-    reference.parentPath.parentPath &&
     t.isCallExpression(reference.parentPath) &&
     reference.parent.callee === reference.node
   ) {
-    isCall = true
     reference.replaceWith(getStyledIdentifier())
+    createStyledComponentPath = reference.parentPath
   }
 
-  if (reference.parentPath && reference.parentPath.parentPath) {
-    const styledCallPath = reference.parentPath.parentPath
+  const styledCallLikeWithStylesPath = createStyledComponentPath
+    ? createStyledComponentPath.parentPath
+    : reference.parentPath.parentPath.isTaggedTemplateExpression()
+    ? reference.parentPath.parentPath
```

Comment:
When will this case happen? If it's a valid styled call, won't `createStyledComponentPath` always be set?

Reply:
It was indeed redundant, some leftover from a WIP - I had to add this to workaround a bug in the code leading to this but it seems that I later fixed the underlying issue. I've cleaned up this redundant code and everything looks OK, thanks for noticing!